run.js

```diff
@@ -99,7 +99,6 @@ const run = async (code, environment, timeout) => {
   if (environment === 'script') {
     const script = new Script(code, {
       filename: FILENAME,
-      timeout,
```

Comment:
there's no timeout in Script constructor https://nodejs.org/api/vm.html#vm_new_vm_script_code_options